runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -2391,6 +2391,12 @@ J9::TransformUtil::refineMethodHandleInvokeBasic(TR::Compilation* comp, TR::Tree
        (symRef->getOwningMethodIndex(), -1, refinedMethod, TR::MethodSymbol::Static);
 
    TR::Node::recreateWithSymRef(node, refinedMethod->directCallOpCode(), newSymRef);
+   // doNotProfile flag is used to imply whether the bytecode can OSR under voluntary OSR. Above transformation
+   // will set this flag to indicate the node cannot OSR. However, it can the transformation does not change
+   // the operand stack and local state before the call.
```

Comment:
> However, it can the transformation does not change the operand stack and local state before the call.

Do you mean to say:

```However, the transformation does not change the operand stack and local state before the call, hence it can OSR.```

If not, can you clarify please?

Reply:
Yes, fixed.